tests/search/search_export_tests.py

```diff
@@ -12,6 +23,86 @@
 # python manage.py test tests.search.search_export_tests --settings="tests.test_settings"
 
 class SearchExportTests(ArchesTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.factory = RequestFactory()
+
+        LanguageSynchronizer.synchronize_settings_with_db()
+        models.ResourceInstance.objects.all().delete()
+        with open(os.path.join("tests/fixtures/resource_graphs/Search Test Model.json"), "r") as f:
+            archesfile = JSONDeserializer().deserialize(f)
+        ResourceGraphImporter(archesfile["graph"])
+
+        cls.search_model_graphid = "d291a445-fa5f-11e6-afa8-14109fd34195"
+        # cls.search_model_cultural_period_nodeid = "7a182580-fa60-11e6-96d1-14109fd34195"
+        # cls.search_model_creation_date_nodeid = "1c1d05f5-fa60-11e6-887f-14109fd34195"
+        # cls.search_model_destruction_date_nodeid = "e771b8a1-65fe-11e7-9163-14109fd34195"
+        cls.search_model_name_nodeid = "2fe14de3-fa61-11e6-897b-14109fd34195"
+        # cls.search_model_sensitive_info_nodeid = "57446fae-65ff-11e7-b63a-14109fd34195"
+        # cls.search_model_geom_nodeid = "3ebc6785-fa61-11e6-8c85-14109fd34195"
+        
+        cls.user = User.objects.create_user("unpriviliged_user", "unpriviliged_user@archesproject.org", "test")
+
+        test_resourceinstanceid = uuid.uuid4()
+
+        cls.loadOntology()
+        cls.ensure_resource_test_model_loaded()
+        models.ResourceInstance.objects.get_or_create(graph_id=cls.search_model_graphid, resourceinstanceid=test_resourceinstanceid)
+        tile_data = {}
+        tile_data[cls.search_model_name_nodeid] = {"en": {
+            "value": "Etiwanda Avenue Street Trees",
+            "direction": "ltr"
+        }}
+        new_tile = Tile(resourceinstance_id=test_resourceinstanceid, data=tile_data, nodegroup_id=cls.search_model_name_nodeid)
+        new_tile.save()
+        time.sleep(1)
```

Comment:
Is this here because a db trigger is going to fire after this? A comment would help, if you don't mind.

Reply:
you're referring to the `time.sleep(1)`? It's because of async indexing and for sure I'll put a comment